packages/react/src/components/Button/Button-story.js

```diff
@@ -35,11 +34,12 @@ const icons = {
 const iconMap = {
   iconAddSolid,
   iconSearch,
-  Add16,
-  AddFilled16,
-  Search16,
-  PlayOutlineFilled16,
-  PlayOutlineFilled32,
+  Add16: Add,
+  AddFilled16: AddFilled,
+  Search16: Search,
+  PlayOutlineFilled16: PlayOutlineFilled,
```

Comment:
this icon looks slightly different in the preview compared to what's on `main`

preview:

![image](https://user-images.githubusercontent.com/8265238/118001800-adb17600-b30c-11eb-88b3-49660085f686.png)

`main`:

![image](https://user-images.githubusercontent.com/8265238/118001833-b6a24780-b30c-11eb-9dde-2c58c558e976.png)